src/commcare_cloud/environment/schemas/terraform.py

```diff
@@ -270,11 +276,12 @@ class awsmqConfig(jsonobject.JsonObject):
     deployment_mode = jsonobject.StringProperty(default="CLUSTER_MULTI_AZ")
     engine_type = jsonobject.StringProperty(default="RabbitMQ")
     engine_version = jsonobject.StringProperty(default="3.10.10")
-    host_instance_type = jsonobject.StringProperty(default="mq.m5.large")   
+    host_instance_type = jsonobject.StringProperty(default="mq.m5.large")
     publicly_accessible = jsonobject.BooleanProperty(default=False)
     logs_general = jsonobject.BooleanProperty(default=True)
-    audit_log_enabled = jsonobject.BooleanProperty(default=False) 
-    encryption_enabled = jsonobject.BooleanProperty(default=False)     
```

Comment:
Looks like these lines had trailing whitespace - not sure why they don't show up as such in the diff

Reply:
Ha yeah I was wondering the same thing